src/ajit/Assignment1.java

```diff
@@ -0,0 +1,60 @@
+package ajit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Assignment1 {
+
+	public static void main(String[] args) throws InterruptedException {
+		String os = System.getProperty("os.name").toLowerCase();
+		String path = os.contains("windows") ? "./resources/windows/chromedriver.exe"
+				: os.contains("mac") ? "./resources/mac/chromedriver" : null;
+
+		System.setProperty("webdriver.chrome.driver", path);
+		WebDriver driver = new ChromeDriver();
+		driver.get("D:\\JAVA_LEARNING\\Projects\\TechnoGitProject\\FirstForm.html");
+		System.out.println("AutomationByKrishna website open successfully.");
+		// Verify Page Title
+		String actualTitle = driver.getTitle();
+		String expectedTitle = "TECHNOCREDITS";
+		if (actualTitle.equals(expectedTitle))
+			System.out.println("Title Matched");
+		else
+			System.out.println("Title didn't match, Actual URL : " + actualTitle);
+		// Select Radio button
+		if (driver.findElement(By.id("maleG")).isSelected()) {
```

Comment:
for Gender radio button if ‘Female’ radio button is by default selected then select ‘Male’   radio button and vice versa.
You have done reverse validation.

Reply:
Incorporated Changes.